setup.cfg

```diff
@@ -12,4 +12,5 @@ filterwarnings =
     ignore:.*ufunc size changed.*:RuntimeWarning
     ignore:Using or importing the ABCs:DeprecationWarning
     ignore:the imp module is deprecated in favour of importlib:DeprecationWarning
+    ignore:.*trait handler has been deprecated.*:DeprecationWarning
```

Comment:
Upstream fix is in https://github.com/enthought/mayavi/pull/896 waiting for merge